Client/Commands.cs

```diff
@@ -21,6 +21,8 @@ public class List
 
             [Option('r', "remote", Required = false, HelpText = "List directories & files on remote servers")]
             public string? Remote { get; set; }
+
+
```

Comment:
Avoid adding unnecessary white space in the future. Just something to be cautious of.